README.md

```diff
@@ -20,6 +20,18 @@ Please use the [ISLE Documentation](https://islandora-collaboration-group.github
 * Time required < 30 minutes.
 * **Windows Users**: Please open the .env and uncomment `COMPOSE_CONVERT_WINDOWS_PATHS=1`
 
+## **AS OF THE ISLE v.1.4.0 release in November 2019 (MANDATORY)**
+
+Due to removal of older files and folders and git changes in the ISLE git repository as of `November 28 2019`, please review the following:
```

Comment:
as of `November 28, 2019`,

Reply:
should this be changed here? I'm not clear on the direction?

Reply:
Changed to today's date?

Reply:
Oh, sorry.  Just the comma after the 28.

Reply:
I'll change it to today's date.